devel/logcat-colorize/Portfile

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+PortSystem          1.0
+name                logcat-colorize
+version             0.8.1
+license             Apache-2
+categories          devel
+platforms           darwin
+maintainers         @carlonluca \
+                    gmail.com:carlon.luca
+description         Parser of the Android Debug Bridge logcat output
+long_description    logcat-colorize is a parser for the output of the \
+                    Android Debug Bridge logcat output. It accepts the \
+                    logcat output from stdin and produces a formatted \
+                    output, optimised for reading in a console.
+homepage            https://github.com/carlonluca/logcat-colorize
+master_sites        https://github.com/carlonluca/logcat-colorize/releases/download/v0.8.1/
+checksums           rmd160  a7f465551403929f73c5b76649a73e043e33105a \
+                    sha256  4f9c8a71b5f90b6448e6f57e7f2349b37513ed8e42878a640403e966b62a4048 \
+                    size    14518
+depends_lib         port:boost
+use_configure       no
+build.target
```

Comment:
Given no configure file, you need to either account for the `+universal` variant or at least disable it.